src/main/java/io/reactivex/Single.java

```diff
@@ -2039,6 +2045,8 @@
 
     /**
      * Delays the emission of the success or error signal from the current Single by the specified amount.
+     * <p>
+     * <img width="640" height="457" src="https://raw.githubusercontent.com/UMFsimke/RxJava/Images/Images/DelaySchedulerIncludingError.png" alt="">
```

Comment:
Please also color the bottom arrows as orange to indicate they get executed on the provided scheduler.